examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_2.ipynb

```diff
@@ -0,0 +1,339 @@
+    "# import urllib.request\n",
+    "# url_path = \"https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv\"\n",
+    "# urllib.request.urlretrieve(url_path, \"taxi.csv\")\n",
+    "# path = \"taxi.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `pandas.read_csv`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "pandas_df = pandas.read_csv(s3_path, parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"])\n",
+    "\n",
+    "end = time.time()\n",
+    "pandas_duration = end - start\n",
+    "print(\"Time to read with pandas: {} seconds\".format(round(pandas_duration, 3)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Expect pandas to take >3 minutes on EC2, longer locally\n",
+    "\n",
+    "This is a good time to chat with your neighbor\n",
+    "Dicussion topics\n",
+    "- Do you work with a large amount of data daily?\n",
+    "- How big is your data?\n",
+    "- What’s the common use case of your data?\n",
+    "- Do you use any big data analytics tools?\n",
+    "- Do you use any interactive analytics tool?\n",
+    "- What’s are some drawbacks of your current interative analytic tools today?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `modin.pandas.read_csv`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "modin_df = pd.read_csv(s3_path, parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"])\n",
+    "\n",
+    "end = time.time()\n",
+    "modin_duration = end - start\n",
+    "print(\"Time to read with Modin: {} seconds\".format(round(modin_duration, 3)))\n",
+    "\n",
+    "print(\"Modin is {}x faster than pandas at `read_csv`!\".format(round(pandas_duration / modin_duration, 2)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Are they equals?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin_df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for exercise: Reduces\n",
+    "\n",
+    "In pandas, a reduce would be something along the lines of a `sum` or `count`. It computes some summary statistics about the rows or columns. We will be using `sum`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "pandas_sum = (pandas_df[:1000000]).sum()\n",
```

Comment:
Summing a part of df rows doesn't have a lot of sense. How are `pandas_df.sum()` and `modin_df.sum()` comparable? Do these take a long time? If so, can we use df.count instead?